samples/exposedapp/src/test/java/io/halkyon/HelmDeploymentE2EIT.java

```diff
@@ -37,21 +37,21 @@ class HelmDeploymentE2EIT {
     final static Logger log = Logger.getLogger(HelmDeploymentE2EIT.class);
 
     static class RunOnlyIfHelmIsAvailableCondition implements ExecutionCondition {
-        private static Boolean helmIsAvailable;
+        private static boolean helmIsAvailable = false;
```

Comment:
I'd rather disable the test explicitly using `@Disabled` but that's fine for now.